File: 1.Arrays and Strings/8.Move Zeros/Solution.js
function moveZeros(nums){
    let i=0;
    for(let j=0; j<nums.length; j++){
        if(nums[j] !== 0){
            [nums[i], nums[j]] = [nums[j], nums[i]];
            i++;
        }
    }
    return nums;
}

let nums = [1,0,1,0,1];
console.log(moveZeros(nums));

File: 1.Arrays and Strings/8.Move Zeros/Solution.java
import java.util.Arrays;

public class Solution {
    public static int[] moveZeros(int[] nums){
        int i=0;
        for(int j=0; j<nums.length; j++){
            if(nums[j] != 0){
                int temp = nums[i];
                nums[i] = nums[j];
                nums[j] = temp;
                i++;
            }
        }
        return nums;
    }

    public static void main(String[] args) {
        int[] nums = {1,0,1,0,1};
        int[] res = moveZeros(nums);
        System.out.println(Arrays.toString(res));
    }
}